icons/jetbrains/jetbrains-plain.svg

```diff
@@ -1 +1,14 @@
+    <path d="M104.64 95.086L108.28 95.086L114.063995 108.799995L110.034996 108.799995L109.06 106.329994L103.795 106.329994L102.82 108.799995L98.92 108.799995L104.64 95.086zM107.955 103.405L106.46 99.57L104.9 103.405L107.955 103.405zM118.419 95.151L114.649 95.151L114.649 108.735L118.419 108.735L118.419 95.151zM119.849 95.151L123.358 95.151L128.298 102.365L128.298 95.151L132.002 95.151L132.002 108.735L128.688 108.735L123.618004 101.325L123.618004 108.735L119.84901 108.735L119.84901 95.151zM132.652 106.72L134.732 104.185C 136.097 105.225 137.592 105.81 139.152 105.81C 140.19199 105.81 140.71199 105.485 140.71199 104.899994L140.71199 104.899994L140.71199 104.83499C 140.71199 104.24999 140.25699 103.98999 138.372 103.53499C 135.44699 102.88499 133.237 102.039986 133.237 99.31099L133.237 99.31099L133.237 99.24599C 133.237 96.71098 135.252 94.89098 138.502 94.89098C 140.842 94.89098 142.596 95.540985 144.091 96.71098L144.091 96.71098L142.20601 99.375984C 140.97202 98.46598 139.60701 98.01099 138.37201 98.01099C 137.462 98.01099 137.007 98.400986 137.007 98.85599L137.007 98.85599L137.007 98.92099C 137.007 99.57099 137.462 99.83099 139.347 100.22099C 142.466 100.870995 144.416 101.909996 144.416 104.44499L144.416 104.44499L144.416 104.509995C 144.416 107.30499 142.206 108.92999 138.892 108.92999C 136.617 108.994995 134.342 108.215 132.652 106.71999z" stroke="none" fill="#FFFFFF" fill-rule="nonzero" />
+</g>
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M0 38.467c0 5.648 3.664 10.418 8.74 12.135l.114.038.343.115 44.458 13.547c.305.115.649.191.992.191a3.16 3.16 0 0 0 3.168-3.167c0-1.145-.61-2.138-1.527-2.71L21.065 28.697a12.638 12.638 0 0 0-8.28-3.053C5.724 25.644 0 31.407 0 38.468v-.001z" fill="#FFF"/><path d="M49.076 1.488 5.99 27.628A12.781 12.781 0 0 0 0 38.468c0 7.06 5.725 12.822 12.784 12.822 3.244 0 6.182-1.221 8.434-3.206l38.58-31.445c.115-.076.192-.152.306-.267 1.985-1.717 3.244-4.236 3.244-7.06A9.3 9.3 0 0 0 54.037 0c-1.794 0-3.512.572-4.961 1.488z" fill="#FFF"/><path d="M44.802 9.311a9.245 9.245 0 0 0 3.243 7.06l.115.115 69.377 58.463a6.242 6.242 0 0 0 4.045 1.45c3.473 0 6.259-2.824 6.259-6.296a6.379 6.379 0 0 0-1.87-4.504L60.677 2.748C58.997 1.068 56.67 0 54.113 0a9.3 9.3 0 0 0-9.311 9.311z" fill="#FFF"/><path d="m117.92 65.066-79.033 38.58a12.731 12.731 0 0 0-6.984 11.411c0 7.06 5.724 12.784 12.784 12.784 2.786 0 5.343-.916 7.442-2.404l73.08-50.144c1.64-1.145 2.708-3.015 2.708-5.152 0-3.473-2.823-6.297-6.258-6.297-1.45 0-2.71.458-3.74 1.222z" fill="#FFF"/><path d="M89.489 38.352H38.352V89.49H89.49V38.352z" fill="#010101"/><path d="M63.12 79.91H43.923v3.206h19.195V79.91zM43.121 51.556l1.413-1.335c.381.458.763.725 1.26.725.533 0 .915-.382.915-1.107v-5.038h2.214v5.076c0 .992-.268 1.718-.802 2.251-.535.535-1.26.802-2.175.802-1.412-.038-2.252-.61-2.825-1.374zm6.755-6.793h6.449v1.87H52.09v1.222h3.816v1.755H52.09v1.26h4.274v1.869h-6.488v-7.975zm9.578 1.947h-2.403v-1.946h6.983v1.945H61.67v6.03h-2.214v-6.03zM44 55.906h3.778c.992 0 1.717.267 2.175.725.306.344.497.763.497 1.298v.038c0 .916-.535 1.488-1.222 1.793.954.305 1.603.878 1.603 1.947v.038c0 1.374-1.107 2.213-3.015 2.213h-3.778v-8.051H44zm4.275 2.443c0-.458-.344-.687-.993-.687h-1.107v1.412h1.069c.649 0 1.03-.23 1.03-.725zm-.764 2.289h-1.373v1.488h1.373c.687 0 1.07-.267 1.07-.725v-.038c.037-.458-.306-.725-1.07-.725zm4.007-4.732h3.549c1.221 0 2.06.305 2.595.84.458.458.725 1.068.725 1.87v.038c0 1.22-.648 2.022-1.64 2.48l1.908 2.786h-2.558l-1.603-2.404h-.725v2.404h-2.214v-8.013h-.037zm3.473 3.816c.725 0 1.183-.381 1.183-.954v-.038c0-.649-.458-.954-1.183-.954h-1.222v1.946h1.222z" fill="#FFF"/><path d="M61.44 55.83h2.137l3.396 8.053h-2.365l-.573-1.45h-3.091l-.573 1.45h-2.29l3.359-8.053zm1.946 4.885-.877-2.252-.916 2.252h1.793zm6.144-4.847h-2.213v7.976h2.213v-7.976zm.84 0h2.06l2.9 4.236v-4.236h2.176v7.976H75.56l-2.977-4.35v4.35H70.37v-7.976zm7.517 6.793 1.222-1.488c.801.61 1.679.954 2.595.954.61 0 .916-.19.916-.534v-.039c0-.343-.267-.496-1.374-.763-1.717-.381-3.015-.878-3.015-2.48v-.038c0-1.489 1.183-2.557 3.091-2.557 1.374 0 2.404.381 3.282 1.068l-1.107 1.565c-.724-.534-1.526-.801-2.251-.801-.534 0-.802.229-.802.496v.038c0 .382.268.534 1.374.763 1.832.382 2.977.992 2.977 2.48v.039c0 1.64-1.298 2.595-3.244 2.595-1.336.038-2.671-.42-3.664-1.298z" fill="#FFF"/></svg>

```

Reply:
@Panquesito7: Do you suggesting me a tool for resizing in the same manner `devicon` reqd.?

Reply:
> @Panquesito7: Do you suggesting me a tool for resizing in the same manner `devicon` reqd.?

[`iloveimg`](https://www.iloveimg.com/resize-image/resize-svg) is a very good tool for resizing SVG icons. You just have to learn a bit how to use it well. 🙂

Reply:
`iloveimg` consider everything under `%`, how to set height & width in `px`?

`devicon` consider everything in `px`, ig! :thinking:

Reply:
Just learnt, this won't happen next time for sure! :+1: